Formula/m/mupdf.rb

```diff
@@ -1,8 +1,8 @@
 class Mupdf < Formula
   desc "Lightweight PDF and XPS viewer"
   homepage "https://mupdf.com/"
-  url "https://mupdf.com/downloads/archive/mupdf-1.22.2-source.tar.gz"
-  sha256 "54c66af4e6ef8cea9867cc0320ef925d561b42919ea0d4f89db5c9ef485bbeb7"
+  url "https://mupdf.com/downloads/archive/mupdf-1.23.3-source.tar.gz"
+  sha256 "1ef9a6409bc0a3271586e1b16f78eb156a579521cd212a124b57c4da8b940aad"
   license "AGPL-3.0-or-later"
   revision 1
```

Comment:
```suggestion
```

Reply:
I rebased and removed this line from the mupdf commit.